### PR TITLE
Drop `useful-forks` feature

### LINKED ISSUE #6397: Drop `useful-forks`?
So they rolled out a new forks page, finally: https://github.blog/changelog/2023-02-24-new-forks-page-view/

Unfortunately [it doesn't yet catch forks of forks](https://github.com/community/community/discussions/40968#discussioncomment-5166009).

<table>
<tr>
	<th><a href="https://github.com/EsotericSoftware/kryonet/forks">Native
	<th><code><a href="https://useful-forks.github.io/?repo=EsotericSoftware/kryonet">useful-forks
<tr>
	<td>Screenshot 4
	<td valign=top>Screenshot 3
</table>

### PR DESCRIPTION
**Pull request by fregante**
- Closes #6397 
- Closes https://github.com/refined-github/refined-github/issues/6405

## Test URLs

- https://github.com/probot/template/blob/master/CODE_OF_CONDUCT.md?rgh-link-date=2022-10-12T08%3A11%3A41Z

## Screenshot

Screenshot 8

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -158,7 +158,7 @@ Thanks for contributing! 🦋🙌
 - [](# "link-to-github-io") [Adds a link to visit the user’s github.io website from its repo.](https://user-images.githubusercontent.com/34235681/152473104-c4723999-9239-48fd-baee-273b01c4eb87.png)
 - [](# "github-actions-indicators") [In the workflows sidebar, shows an indicator that a workflow can be triggered manually, and its next scheduled time if relevant.](https://user-images.githubusercontent.com/46634000/139128320-78eb66c7-d485-46c0-bde2-50e00ba989f3.png)
 - [](# "quick-repo-deletion") [Lets you delete your repos in a click, if they have no stars, issues, or PRs.](https://user-images.githubusercontent.com/1402241/99716945-54a80a00-2a6e-11eb-9107-f3517a6ab1bc.gif)
-- [](# "useful-forks") [Helps you find the most active forks](https://user-images.githubusercontent.com/38117856/107463541-542e8500-6b2c-11eb-8b25-082f344c1587.png), via https://useful-forks.github.io.
+- [](# "fork-notice") [Helps you find forks on archived repos.](https://user-images.githubusercontent.com/1402241/230362566-12493c80-bffe-4c7a-b9ba-4a11b1358ab0.png)
```
Comment (fregante) [inline readme.md:161]: Name change because:

- it no longer uses `useful-forks`

File: source/features/useful-forks.tsx [hunk 2]
```diff
@@ -1,78 +0,0 @@
...
-	container!.prepend(
-		<a className="btn mb-2 float-right" href={getUrl()}>
-			<RepoForkedIcon className="mr-2"/>
-			Find useful forks
-		</a>,
-	);
-}
-
-function addLinkToBanner(label: HTMLElement): void {
-	if (label.textContent!.trim() !== 'Public archive') {
```
Comment (fregante) [inline source/features/useful-forks.tsx:39]: Unfortunately this piece of information is no longer available on the page

**Comment by fregante** [comment]
For ease of documentation, I'm just dropping `useful-forks` here and then add the new feature separately

**Comment by stdedos** [comment]
I assume there's no way, for people that actually like that little button, to keep having it after this, right?

**Comment by fregante** [comment]
There's a way: https://chrome.google.com/webstore/detail/useful-forks/aflbdmaojedofngiigjpnlabhginodbf

**Comment by stdedos** [comment]
> Also part of the feature still exists here, but it uses GitHub's own view:
> 
> * [Add `archive-forks-link` feature #6481](https://github.com/refined-github/refined-github/pull/6481)

That focuses on archived repositories. Sometimes, I browse still-active repos.

**Comment by fregante** [comment]
Right, the feature was two-parts:

- show a link on the forks view
- show a link on the archived repo

Since the link now points *to the forks view*, then the first part doesn't make sense, does it?

If you prefer the "Useful Forks" website, then use its extension.

**Comment by fregante** [comment]
You can ask on their repo: 

https://github.com/useful-forks/useful-forks.github.io

Keep in mind that this feature was removed because GitHub's own fork view has been improved and it's just as good as Useful Forks, it shouldn't really be needed:

- https://github.blog/changelog/2023-02-24-new-forks-page-view/

Refer to the original issue here for more info:

- #6397

**Comment by payne911** [comment]
> This is the main reason I used this add-on. :( There is no Firefox add-on for finding useful forks anymore?

@endolith `useful-forks` would be happy to get a PR that would allow Firefox users to use the service.

**Comment by endolith** [comment]
> useful-forks would be happy to get a PR that would allow Firefox users to use the service.

@payne911 I would be happy for you to get that PR, too.  :)  In the meantime, can it be restored to Refined Github?  https://github.com/refined-github/refined-github/issues/6397#issuecomment-1476430935 said "Yeah we can keep it until GitHub's feature is good enough", yet it doesn't appear to be good enough yet.

**Comment by fregante** [comment]
> can it be restored to Refined Github?

No 

> it doesn't appear to be good enough yet.

It is as far as Refined GitHub is concerned.

**Pull request merged**